ui/mirage/scenarios/default.js

```diff
@@ -602,6 +609,7 @@ function allNodeTypes(server) {
 
 function everyFeature(server) {
   server.createList('agent', 3, 'withConsulLink', 'withVaultLink');
+  server.createList('node-pool', count);
```

Comment:
what's `count` in this context?

Reply:
Ops, nice catch. It's a leftover from a previous implementation approach 😅